src/main/kotlin/minesweeper/entity/MineField.kt

```diff
@@ -20,7 +20,52 @@ class MineField(
 
     fun countAroundMines(coordinate: Coordinate): Int {
         return coordinate.adjacentCoordinates()
-            .filter { it.isWithinBounds(width.value, height.value) }
+            .filter { it.isWithinBounds(width, height) }
             .count { _cells.findCell(it) is Cell.Mine }
     }
+
+    fun open(coordinate: Coordinate) {
+        val cell = findCell(coordinate)
```

Comment:
아래와 같이 _cells 프로퍼티로부터 바로 findCell해도 됐을 것 같은데, 따로 메서드로 추출하신 이유가 있으실까요? 🤔
개인적으로는 조금 불필요한 메서드 추출로 보여서요

```suggestion
        val cell = _cells.findCell(coordinate)
```

Reply:
MineFieldConverter에서 해당 메서드를 쓰고 있습니다! 
내부에서는 추출한 메서드를 쓸 필요는 없겠네요